code/modules/antagonists/cult/runes.dm

```diff
@@ -875,7 +875,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 			if(user.stat != CONSCIOUS || HAS_TRAIT(new_human, TRAIT_CRITICAL_CONDITION))
 				break
 			user.apply_damage(0.1, BRUTE)
-			sleep(1)
+			sleep(1 TICKS)
```

Comment:
uhm

Reply:
sleep(1) = wait `1 ds`
sleep(1 TICKS) = `1 * (10/world.fps)`
world.fps on our servers is 20
ergo this went from applying damage to the person every 1 ds to every 0.5ds.

this is absolutely a behavior change.

Reply:
Hey that's my line

Reply:
> Hey that's my line

Hah, was reviewing all the changes before submitting. Glad we both caught it at least. :P

Reply:
i blanked the fact that it was a while loop